### ISSUE TITLE
Reduce ambiguity of imported functions

### ISSUE THREAD

**Issue by sambnh** [Dustpancake/Asteroid#38]
The majority of the imports exist in the form
```python
from modu import func
[...]
x = sqrt(4)  # sqrt may be part of modu, if not redefined in between
```
Instead of the more explicit and readable
```python
import modu
[...]
x = modu.sqrt(4)  # sqrt is visibly part of modu's namespace
```
A more in depth explanation on benefits can be found [here](https://docs.python-guide.org/writing/structure/#modules)
I mostly got fed up with this when trying to name stuff related to the /request endpoint when 'request' is already used by flask

**Comment by JrTheDragon01** [comment]
@sambnh you mentioned this with respect to JSConfig import in run.py; fixed in this specific case as of 3b6fd17677ddf6b8f06c8c773f0390b29ce712e6 :)

**Comment by Dustpancake** [comment]
I think our code is unambigious enough as it is; short classes and relevant namespaces have already been established (see refactor branch soon to be merged). Closing.
